test/e2e/network/dns.go

```diff
@@ -88,6 +89,27 @@ var _ = common.SIGDescribe("DNS", func() {
 		validateDNSResults(ctx, f, pod, append(wheezyFileNames, jessieFileNames...))
 	})
 
+	/*
+		Release: v1.30
+		Testname: DNS, cluster
+		Description: When a Pod is created, the container hostname should match with spec hostname after setting the hostNetwork to true
```

Comment:
nit:  s/When a Pod is created, the container hostname should match with spec hostname after setting the hostNetwork to true/When a Pod is created, with hostNetwork set to true, the container hostname should match with spec hostname.